code/game/objects/items/weapons/melee/energy.dm

```diff
@@ -185,8 +185,5 @@
 /obj/item/weapon/melee/energy/blade/dropped()
 	qdel(src)
 
-/obj/item/weapon/melee/energy/blade/proc/throw()
-	qdel(src)

```

Comment:
well, it was used for NInja so when he threw is energy sword, it would delete it instead of...actually throwing it.